wagtail/wagtail_hooks.py

```diff
@@ -14,7 +14,6 @@
 from wagtail.rich_text.pages import PageLinkHandler
 from wagtail.utils.timestamps import parse_datetime_localized, render_timestamp
 
-
 def require_wagtail_login(next):
```

Comment:
Just one more clean up needed here, re-add this line.

Reply:
Is this a blank line that's to be added here?

Reply:
Just a test for a button is enough